docs/getting_started/installation.rst

```diff
@@ -146,11 +146,14 @@ also use ``pip``.
 
 .. code-block:: bash
 
-  pip install git+https://github.com/modin-project/modin
+  pip install "modin[all] @ git+https://github.com/modin-project/modin"
 
 This will install directly from the repo without you having to manually clone it! Please be aware
 that these changes have not made it into a release and may not be completely stable.
 
+If you would like to install Modin with a specific engine, you can use ``modin[ray]`` or ``modin[dask]`` instead of ``modin[all]`` in the command above.
+For more documentation about installing particular engines, see the `README PyPI installation instructions`_.
```

Comment:
```suggestion
For more documentation about installing particular engines, see the `PyPI installation instructions`_ in Modin's README.
```

Reply:
or even just remove the sentence - For more documentation about installing particular engines, see the `README PyPI installation instructions`_.

Reply:
Because that is a paragraph related to installing Modin from source.